GOVERNANCE.MD

```diff
@@ -1,6 +1,92 @@
 # Community Committee Governance
 
-The Community Committee governance is currently under pull request as the Community Committee Charter.
+Section 1. Guiding Principle.
+
+The Community Committee will operate transparently, openly, collaboratively, and ethically. Project proposals, timelines, and status must not merely be open, but also easily visible to the entire ecosystem.
+
+Section 2. Evolution of Node.js Foundation Governance.
```

Comment:
i'd like to see something here that says what power (if any) this committee has to change how the TSC operates (e.g. CoC changes, moderation)